lib/rototiller/task/collections/command_collection.rb

```diff
@@ -0,0 +1,14 @@
+require 'rototiller/task/collections/param_collection'
+require 'rototiller/task/params/command'
+
+module Rototiller
+  module Task
+
+    class CommandCollection < ParamCollection
+      def allowed_class
+        Command
+      end
```

Comment:
yeah, i also don't like that.  it means whatever uses CommandCollection has to know what class it allows.  it should know itself.
oh.  maybe you mean it's a reader, and the initializer in here sets it, then calls super?
yeah, seems like more hoops to jump through without gaining anything.  this is just an explicit reader.  
can you set a value when doing readers?
`attr_reader :allowed_class = Command`
nope:
```
SyntaxError: (irb):5: syntax error, unexpected '=', expecting keyword_end
attr_reader :other = 'value'
```